docs/get_float.adoc

```diff
@@ -11,27 +11,44 @@ get_float - prompts user for a line of text from stdin and returns the equivalen
 
 *#include <cs50.h>*
 
-*float get_float(string prompt);*
+*float get_float(const string format, ...);*
```

Comment:
Am inclined to change signatures to `const char * format`, so that our man pages are more-comfy, just like libc's. And then in reference50, we can simplify that to `string format`?